model/createRecipeTestSample.json

```diff
@@ -0,0 +1,11 @@
+
+{
+ "user_id":15,
+ "recipe_name":"Tomato pesto chicken pasta",
+ "cuisine":"Italian",
```

Comment:
I think cuisine should probably be an id as well

Reply:
done!